static/app/components/forms/fields/projectMapperField.tsx

```diff
@@ -59,6 +59,20 @@ export class RenderField extends Component<RenderProps, State> {
     selectedMappedValue: null,
   };
 
+  componentDidUpdate(prevProps: RenderProps) {
+    const prevProjectIds = prevProps.sentryProjects.map(proj => proj.id);
+    const projectIds = this.props.sentryProjects.map(proj => proj.id);
+
+    if (prevProjectIds !== projectIds) {
```

Comment:
you can't compare arrays this way

Reply:
maybe use lodash difference or differenceWith https://lodash.com/docs/4.17.15#difference

Reply:
had the correct array comparison locally but didn't save it, will also use `difference`